subsys/logging/Kconfig

```diff
@@ -256,6 +256,19 @@ config LOG_BUFFER_SIZE
 	help
 	  Number of bytes dedicated for the logger internal buffer.
 
+config LOG_STRDUP_MAX_STRING
+	int "Longest string that can be duplicated using log_strdup()"
+	default 32
```

Comment:
In networking side this is too short when we are printing IPv6 addresses. Could this be increased to 46 in that case? The reason being that max length of a printable IPv6 address is 46 bytes (this is the value of INET6_ADDRSTRLEN symbol from Posix).
For example we could have here `default 46 if NETWORKING`

Reply:
done